great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -290,6 +290,12 @@ def get_domain_records(
         Returns:
             A DataFrame (the data on which to compute)
         """
+        table = domain_kwargs.get("table", None)
+        if table:
+            raise ValueError(
+                "PandasExecutionEngine does not currently support multiple named tables."
+            )
+
```

Comment:
Do we ever use `table` again? If not, can we just say `if "table" in domain_kwargs` to simplify this?

Reply:
@cdkini We need to do it this way, because the `table` key is allowed, but its value cannot be any other than `None`.